src/mappings/comptroller.ts

```diff
@@ -44,33 +57,41 @@ export function handleMarketExited(event: MarketExited): void {
 }
 
 export function handleNewCloseFactor(event: NewCloseFactor): void {
+  log.info('COMPTROLLER::handleNewCloseFactor', [])
   let comptroller = Comptroller.load('1')
   comptroller.closeFactor = event.params.newCloseFactorMantissa
   comptroller.save()
 }
 
 export function handleNewCollateralFactor(event: NewCollateralFactor): void {
-  let market = Market.load(event.params.cToken.toHexString())
+  log.info('COMPTROLLER::handleNewCollateralFactor', [])
+  let marketId = event.params.cToken.toHex()
+  let market = Market.load(marketId)
+  if (market == null) {
+    market = createMarket(marketId)
+  }
   market.collateralFactor = event.params.newCollateralFactorMantissa
     .toBigDecimal()
-    .div(mantissaFactorBD)
+    .div(MANTISSA_FACTOR_BD)
```

Comment:
lets avoid renaming variables unless absolutely needed as we are taking changes to a fork and not writing it from scratch? wdyt?

Reply:
I did it for easier code readability. consts def using caps